storybook/src/styles/docs.css

```diff
@@ -67,9 +67,11 @@
 }
 
 .ams-docs-backdrop {
-  background-color: #0006;
+  background-color: var(--ams-dialog-backdrop-background-color);
   block-size: 100%;
   inline-size: 100%;
+  padding-block: 1rem;
+  padding-inline: 1rem;
```

Comment:
This padding makes the container overflow on Chrome, I don't think we want that?

Reply:
We don’t. I applied border-box sizing.